Godeps/Godeps.json

```diff
@@ -2350,82 +2348,82 @@
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/cgroups/fs",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/cgroups/systemd",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/configs",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/configs/validate",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/criurpc",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/intelrdt",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/keys",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/mount",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/seccomp",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/stacktrace",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/system",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/user",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
 			"Rev": "595bea022f077a9e17d7473b34fbaf1adaed9e43"
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runc/libcontainer/utils",
-			"Comment": "v1.0.0-rc4-221-g595bea02",
+			"Comment": "v1.0.0-rc4-221-g595bea0",
```

Comment:
This is updating a lot more than `sys/unix`. Are all these other changes intentional? These hashes look suspicious.

Reply:
Added another significant digit. Not sure if that's a git thing or a godep thing

Reply:
Yeah - the SHAs are same, it is just updating `Comment` field too, not the actual `Rev` field. I am not sure if it was because of git or godep either. :(

Reply:
I can probably manually drop these hunks and just keep the bits I need. Although - I am not sure if full restore, edit and save will again cause these `Comment` fields to be update for someone.